DSCResource.Tests

```diff
@@ -0,0 +1 @@
+Subproject commit b367d4beaaa93c8c2f5871a71fcf5e1090c8a032
```

Comment:
We shouldn't be adding DSCResource.Tests as a submodule. It's a hassle to keep the references up to date and our test automation in appveyor clones the repo automatically anyways.